FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/ConvertUnits.java

```diff
@@ -1,10 +1,30 @@
 package hec.ensemble.stats;
+import tech.units.indriya.function.RationalConverter;
+import javax.measure.IncommensurableException;
+import javax.measure.Unit;
+
+import static javax.measure.MetricPrefix.KILO;
+import static systems.uom.common.USCustomary.ACRE_FOOT;
+import static systems.uom.common.USCustomary.CUBIC_FOOT;
+import static tech.units.indriya.unit.Units.SECOND;
 
 public final class ConvertUnits {
     private ConvertUnits() {
     }
 
-    public static float convertCfsAcreFeet(int seconds) {
-        return (float) seconds / 43560;
+    public static Unit<?> convertStringUnits(String units) {
+        return UnitsUtil.convert(units);
+    }
+
+    public static double getAccumulationConversionFactor(Unit<?> unit) throws IncommensurableException {
```

Comment:
This method can be declared as package private.

Reply:
Changed to package private